examples/experimental/openvino/activation_sparsity_statistic/main.py

```diff
@@ -0,0 +1,80 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import argparse
+from pathlib import Path
+
+import openvino.runtime as ov
+import torch
+from torchvision import datasets
+from torchvision import transforms
+
+import nncf
+from nncf.experimental.openvino_native.activation_sparsity_statistic.activation_sparsity_statistic import \
+    activation_sparsity_statistic_impl
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        prog="Activation sparsity statistic for ResNet models.",
+        description="Collect activation sparsity statistic and write statistic to the model.",
+    )
+    parser.add_argument("-m", "--model_path", help="Target OpenVINO model (.xml).")
+    parser.add_argument("-d", "--data_path", help="Path to folder with images.")
+    return parser.parse_args()
+
+
+def run_example(model_path, data_path):
+    # Step 1: Initialize OpenVINO model.
+    ir_model_xml = Path(model_path)
+    ir_model_bin = ir_model_xml.with_suffix(".bin")
```

Comment:
You can specify the path to .xml only. It can reduce the number of lines from 3 to 1 here.